wwwroot/cgi-bin/lib/robots.pm

```diff
@@ -2551,8 +2576,8 @@
 'spider','spider',
 'sucker','sucker',
 'validator','validator',
-'bot[\s_+:,\.\;\/\\\-]','Unknown robot identified by bot\*',
-'[\s_+:,\.\;\/\\\-]bot','Unknown robot identified by \*bot',
+'bot[\s_+:,\.;/\\-]','Unknown robot identified by bot\*',
+'[\s_+:,\.;/\\-]bot','Unknown robot identified by \*bot',

```

Comment:
Here the / is inside the [ ] , so we should keep the \/  instead of  /  to detect char /, shouldn't it ?